include/seqan3/io/structure_file/format_vienna.hpp

```diff
@@ -129,9 +129,9 @@ class structure_file_format_vienna
               comment_type & SEQAN3_DOXYGEN_ONLY(comment),
               offset_type & SEQAN3_DOXYGEN_ONLY(offset))
     {
-        auto stream_view = view::subrange<decltype(std::istreambuf_iterator<char>{stream}),
-                                          decltype(std::istreambuf_iterator<char>{})>
-                           { std::istreambuf_iterator<char>{stream}, std::istreambuf_iterator<char>{} };
+        using stream_t = std::istreambuf_iterator<typename stream_type::char_type>;
```

Comment:
It's not the stream type but the stream iterator type, maybe call it `stream_it_t`?

Reply:
yes, good point!